src/app/core/store/shopping/product-reviews/product-reviews.reducer.ts

```diff
@@ -0,0 +1,31 @@
+import { EntityState, createEntityAdapter } from '@ngrx/entity';
+import { createReducer, on } from '@ngrx/store';
+
+import { HttpError } from 'ish-core/models/http-error/http-error.model';
+import { ProductReviews as ProductReviewsModel } from 'ish-core/models/product-reviews/product-reviews.model';
+import { setErrorOn } from 'ish-core/utils/ngrx-creators';
+
+import { loadProductReviews, loadProductReviewsFail, loadProductReviewsSuccess } from './product-reviews.actions';
+
+export const productReviewsAdapter = createEntityAdapter<ProductReviewsModel>({
+  selectId: review => review.sku,
+});
+
+export interface ProductReviewsState extends EntityState<ProductReviewsModel> {
+  loading: boolean;
+  error: HttpError;
+}
+
+const initialState: ProductReviewsState = productReviewsAdapter.getInitialState({
+  loading: false,
+  error: undefined,
+});
+
+export const productReviewsReducer = createReducer(
+  initialState,
+  on(loadProductReviews, state => ({ ...state, loading: true })),
```

Comment:
pls use the methods setLoadingOn, unsetLoadingAndErrorOn, 

make sure, loading is set by the load action, and unset by the fail and success action and the error is reseted at least by the next success action